jdi-light/src/main/java/com/epam/jdi/light/elements/composite/WebPage.java

```diff
@@ -455,14 +477,20 @@ public boolean contains() {
             return equals == null || equals.equals("") || value.contains(equals);
         }
     }
+
+    public static PageChecks CHECK_AFTER_OPEN = PageChecks.NONE;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5408219)